questions/qa-bidi-unicode-controls.en.html

```diff
@@ -118,7 +125,7 @@ <h2>Changing the base direction</h2>
           <td>LEFT-TO-RIGHT EMBEDDING</td>
           <td>U+202A</td>
           <td>dir = &quot;ltr&quot;</td>
```

Comment:
Why adding spaces around the equal sign? I think if the reader copies the code to other places, it won't work.